lib/modules/apostrophe-images/lib/cursor.js

```diff
@@ -40,6 +40,65 @@ module.exports = {
       }
     });
 
+    self.addFilter('orientation', {
+      finalize: function() {
```

Comment:
You should include `def: null` for clarity: if the filter is not explicitly used all orientations are shown.

Reply:
ok done